packages/ui/src/contexts/SwapProvider/useEvmSwap.test.tsx

```diff
@@ -55,6 +56,9 @@ jest.mock('../ConnectionProvider', () => ({ useConnectionContext: jest.fn() }));
 jest.mock('../FeatureFlagsProvider', () => ({
   useFeatureFlagContext: jest.fn(),
 }));
+jest.mock('../NetworkFeeProvider', () => ({
+  useNetworkFeeContext: jest.fn(),
+}));
```

Comment:
jest.mock automatically mocks all members of the module, so the 2nd argument can be skipped here
```suggestion
jest.mock('../NetworkFeeProvider');
```